reporter/src/funTest/kotlin/reporters/freemarker/PlainTextTemplateReporterFunTest.kt

```diff
@@ -86,13 +85,11 @@ class PlainTextTemplateReporterFunTest : WordSpec({
 private fun TestConfiguration.generateReport(
     ortResult: OrtResult,
     config: OrtConfiguration = OrtConfiguration(),
-    copyrightGarbage: CopyrightGarbage = CopyrightGarbage(),
     options: Map<String, String> = emptyMap()
 ): String {
     val input = ReporterInput(
         ortResult,
         config,
-        copyrightGarbage = copyrightGarbage,
```

Comment:
Commit message: This is not "inlining". It would be inlining if this line was written as `copyrightGarbage = CopyrightGarbage()`. What you're doing is to remove a parameter that's always set to the default value.